### ISSUE TITLE
Grammar detector

### ISSUE THREAD

**Issue by antonmedv** [phikiphp/phiki#40]
Hi, amazing library! 🔥 Very cool. I used it on my blog.

One request is to add a helper to detect proper grammar from a string:
* js => javascript
* jsx => javascript
* yaml => yaml
* yml => yml

**Comment by antonmedv** [comment]
PS Cool logo! ;)

**Comment by ryangjchandler** [comment]
@antonmedv Yeah this is something I've thought about. Most grammars have some sort of "first-line pattern" to determine if a grammar can be used, so that's probably how this would work.

I'll need to spend some time implementing it though.
